Services/YoutubeApiService.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+/*
+ * Youtube Data API v3 - Simple Client
+ *
+ * @see https://developers.google.com/youtube/v3/docs/channels/list
+ * @see https://developers.google.com/youtube/v3/docs/playlistItems/list
+ */
+
+use Illuminate\Support\Collection;
+
+class YoutubeApiService
+{
+    private const API_BASE_URL = 'https://youtube.googleapis.com/youtube/v3/';
+    private string $apiKey;
+
+    public function __construct()
+    {
+        $this->apiKey = config('services.youtube.key');
+    }
+
+    public function getChannelInformation(
+        string $channelId = 'UC_x5XG1OV2P6uZZ5FSM9Ttw'
+    ): array
+    {
+        //compose endpoint url for API
+        $apiUrl = sprintf(
+            '%s%s?part=%s&id=%s&key=%s',
+            self::API_BASE_URL,
+            'channel',
```

Comment:
I'm not sure, but I guess the endpoint has to be `channels` according to the API documentation